### ISSUE TITLE
Dependency charcode is not installed during angel_framework installation

### ISSUE THREAD

**Issue by alexd1971** [angel-dart/framework#160]
pubspec.yaml:
```
...
dependencies:
  angel_framework: ^1.0.8
```
```
$ pub get
Resolving dependencies... (4.5s)
+ angel_framework 1.0.8
+ angel_model 1.0.0
+ angel_route 1.0.5
+ body_parser 1.0.2
+ container 0.1.2
+ flatten 1.0.0
+ http_server 0.9.6
+ json_god 2.0.0-beta+1
+ merge_map 1.0.0
+ meta 1.1.1
+ mime 0.9.3
+ path 1.4.2
+ random_string 0.0.1
Downloading angel_framework 1.0.8...
Downloading body_parser 1.0.2...
Downloading angel_route 1.0.5...
Changed 13 dependencies!
```
Starting server (bin/server.dart):
```
import 'dart:io';

import 'package:angel_framework/angel_framework.dart';

main () async {
  var app = new Angel();
  var server = await app.startServer(InternetAddress.LOOPBACK_IP_V4, 1337);
  print('Listening at http://${server.address.address}:${server.port}');
}
```
Console output:
```
$ dart bin/server.dart
Unhandled exception:
Could not import "package:charcode/charcode.dart" from "package:angel_framework/src/http/server.dart": No mapping for 'charcode' package when resolving 'package:charcode/charcode.dart'.
null
```
Adding chracode manually:
```
dependencies:
  angel_framework: ^1.0.8
  charcode: "^1.1.1"
```
```
$ pub get
Resolving dependencies...
+ charcode 1.1.1
Changed 1 dependency!
```
Starting server:
```
$ dart bin/server.dart
Listening at http://127.0.0.1:1337
```

**Comment by thosakwe** [comment]
Thanks! I could have sworn I had it as a dependency.

**Comment by thosakwe** [comment]
Closed in `1.0.9`. Writing some tests for https://github.com/angel-dart/framework/issues/159, then will publish. Thanks for noticing this bug!
